Sources/web3swift/Web3/Web3+JSONRPC.swift

```diff
@@ -44,13 +44,11 @@ public struct JSONRPCrequest: Encodable {
     }
 
     public var isValid: Bool {
-        get {
-            if self.method == nil {
-                return false
-            }
-            guard let method = self.method else {return false}
-            return method.requiredNumOfParameters == self.params?.params.count
+        if self.method == nil {
+            return false
         }
+        guard let method = self.method else {return false}
```

Comment:
`if` and `guard` statements are the same.
`guard` will be enough.

Reply:
inherited code, but I'll fix this one, thanks